### PR TITLE
Bumper release notes

### PR DESCRIPTION
**Pull request by aludvik**

### CODE REVIEW

File: generator/source/index.md [hunk 1]
```diff
@@ -23,6 +23,15 @@ community and documented
 Information on what _Active_ entails can be found in the
 [Hyperledger Project Lifecycle document](https://wiki.hyperledger.org/community/project-lifecycle).
 
+# Latest Release
+
+The latest release of Hyperledger Sawtooth is 1.1, codename Bumper.
```
Comment (annechenette) [inline generator/source/index.md]: Is it bumper or Bumper?

Also, it should be codenamed (not codename).

File: generator/source/release/bumper/notes.md [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+
+## New Features
+
+- The consensus API has been completely redesigned. Consensus has been moved to
+  a separate process, called a “consensus engine”. See [Hyperledger Sawtooth
+  Consensus](#hyperledger-sawtooth-consensus), below for more details.
+- State pruning is now supported at a configurable block horizon. This helps
+  limit the total storage requirements for global state by removing historic
+  state after the configured horizon. Check out the [state pruning
+  RFC](https://github.com/hyperledger/sawtooth-rfcs/pull/8) for more details:
```
Comment (dcmiddle) [inline generator/source/release/bumper/notes.md]: :s/:/.

**Review by vaporos — APPROVED** [review]

**Review by dcmiddle — CHANGES_REQUESTED** [review]

**Comment by aludvik** [comment]
@dcmiddle fixed punctuation typo

**Review by dcmiddle — APPROVED** [review]

**Comment by aludvik** [comment]
Rebased on master to pick up Jenkinsfile

**Comment by aludvik** [comment]
Wait until release is live to merge

**Review by annechenette — COMMENTED** [review]
The documentation section appears incomplete - it appears that the only doc changes were to add a Rust SDK and a glossary.  I can provide an additional commit with the formatting changes.

**Comment by aludvik** [comment]
@chenette I think it looks cleaner with the sub-headings.

**Sub-headings**
before

**Sub-lists**
after

**Comment by annechenette** [comment]
I suspect that's because the sub-bullets (dashes) aren't rendering correctly in this view.  I assume that the indenting would be correct on the website, because my online markdown viewers assure me that the sub-bullet indents are correct. Also, it's identical to the format used in the Consensus section.

**Comment by annechenette** [comment]
I stand corrected.  I need more spaces for my indented lists.  I shall push a fix for the Documentation section, then follow with a fix for the Consensus section.

> To nest one list within another, indent each item in the sublist by four spaces.

**Comment by annechenette** [comment]
@aludvik - I wasn't able to get nested (indented) lists working in the "build artifacts". I don't know if this is just an issue with the build environment, but I don't want to risk introducing these format errors in the website.  So I pushed a fixup commit that makes the subheading approach consistent  by changing the first two bullets to subheads. I hope this is an acceptable compromise.

**Review by vaporos — APPROVED** [review]

**Pull request closed without merging**